docs/kratos/organizations/organizations.mdx

```diff
@@ -3,6 +3,14 @@ id: organizations
 title: Organizations and B2B Single Sign-On
 ---
 
+:::info
```

Comment:
btw https://github.com/ory/docs/pull/1708#discussion_r1568701196 

IMO both is fine 🤷

Reply:
Oh, I thought that was a mistake, because it is really a general remark about this feature, and an upsell toggle. WDYT, move it back?